src/routes/(admin)/tests/test-[type]/[id]/Configuration.svelte

```
@@ -351,6 +374,50 @@
+
+									{#each formsOptions as form (form.id)}
+										<Select.Item value={form.id}>
+											{form.name}
+										</Select.Item>
+									{/each}
+								</Select.Group>
+							</Select.Content>
+						</Select.Root>
+
+						<a
+							href="/forms/add/new"
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)